src/snowflake/snowpark/mock/_analyzer.py

```diff
@@ -730,18 +730,20 @@ def do_resolve_with_resolved_children(
             )
 
         if isinstance(logical_plan, CopyIntoLocationNode):
-            return self.plan_builder.copy_into_location(
-                query=resolved_children[logical_plan.child],
-                stage_location=logical_plan.stage_location,
-                partition_by=self.analyze(logical_plan.partition_by, expr_to_alias)
-                if logical_plan.partition_by
-                else None,
-                file_format_name=logical_plan.file_format_name,
-                file_format_type=logical_plan.file_format_type,
-                format_type_options=logical_plan.format_type_options,
-                header=logical_plan.header,
-                **logical_plan.copy_options,
-            )
+            return MockExecutionPlan(logical_plan, self.session)
+
```

Comment:
can remove this comment here, issue is that for some reason a SnowflakePlan (not MockPlan) is created here. Likely a copy&paste error.